### DISCUSSION TITLE
OCI SB 5.1 Provision/DataPipeline/Kafka Job fails

### DISCUSSION THREAD

**Discussion by subhashchandrab** [Sunbird-Ed/Community#409] [Installation]
Following the documentation https://ed.sunbird.org/use/installing-ed-platform/release-5.1.0/build. The job Provision/DataPipeline/Kafka failing with below error message. Does it mean any prerequisite is missing? Where is this service monit get installed. 

TASK [service] *****************************************************************
fatal: [10.90.8.104]: FAILED! => {
    "changed": false
}

MSG:

Could not find the requested service monit: host

...ignoring
fatal: [10.90.8.106]: FAILED! => {
    "changed": false
}

MSG:

Could not find the requested service monit: host

...ignoring
fatal: [10.90.8.105]: FAILED! => {
    "changed": false
}

MSG:

Could not find the requested service monit: host

**Comment by beepdot** [comment]
@subhashchandrab Can you post the complete console logs?
